Python/Calculator/INSTRUCTIONS.md

```diff
@@ -0,0 +1,38 @@
+# INSTRUCTIONS
+
+- Press RUN
+- Enter first number
+- click enter button on the keyboard
+- enter operator
+- click enter button 
+- enter the second number
+- click enter button
+
+
+## FOR OPERATORS ENTER THE FOLLOWING
+
+you can use upper or lower case for operations    
+
+Operators|Functions
```

Comment:
The readme needs proper alignment right now it's all messed up:
![image](https://user-images.githubusercontent.com/43691873/95007881-50e13500-0632-11eb-8142-a9e654e00619.png)